### PR TITLE
feature/1191 - Consolidate command bar buttons

### LINKED ISSUE #1191: Consolidate command bar buttons
We are accumulating a lot of buttons in the command bar, and they should be consolidated into groups so they take up less room, are more organized, and more intuitive to figure out what the icons are supposed to represent because you have additional context.

```
Scope --> [SIDs, STARs, terrain, video map, navaids, restricted, range rings]
Settings
Help --> [Tutorial, GitHub link, YouTube link, Slack link, Command Reference link, Airport Guide link]
```

### PR DESCRIPTION
**Pull request by manuel3108**
Resolves #1191 

The purpose of this pull request is to improve the controls by grouping buttons

### CODE REVIEW

File: src/assets/style/module/controls/_controls.less [hunk 1]
```diff
@@ -1,46 +1,113 @@
...
+    margin-left: 0;
+}
+
+.dropup-content {
+    display: none;
+    position: absolute;
+    background-color: @dark-gray;
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+    bottom: @command-bar-height;
+    float: right;
```
Comment (n8rzz) [inline src/assets/style/module/controls/_controls.less]: float won't apply on an absolutely positioned element

File: src/templates/_footer.hbs [hunk 2]
```diff
@@ -1,59 +1,98 @@
...
-        <li class="control pause-toggle" title="Pause simulation">
-            <img src="assets/images/icons/pause.png" />
-        </li>
-        <li class="control toggle-speech" title="Enable speech synthesis (experimental)">
-            <img src="assets/images/icons/speech.png" />
-        </li>
-        <li class="control toggle-traffic" title="Adjust traffic rate">
-            <img src="assets/images/icons/traffic.png" />
+
+    <ul>
```
Comment (n8rzz) [inline src/templates/_footer.hbs]: this should still use a class name.  directly styling elements has a different specificity that styling classnames (https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), and we prefer to style with classnames.

File: src/templates/_footer.hbs [hunk 3]
```diff
@@ -1,59 +1,98 @@
...
+        <li class="dropup">
```
Comment (n8rzz) [inline src/templates/_footer.hbs]: hmm, we may want to generalize this name a bit.  this item doesn't actually include a `dropup` element, it's just a menu item

Comment (manuel3108) [reply src/templates/_footer.hbs]: jup, agree. Do you think that it's better now?

**Comment by n8rzz** [comment]
It might make a little more sense to have this menu item last in the list

Screen Shot 2019-11-02 at 12 33 24 PM

**Comment by manuel3108** [comment]
I have done the reorder. Where do you want to have the airpot-guide button?
I put it beside the switch-airport button, because it doesn't fit in any of those subcategories in my opinion.

File: package-lock.json [hunk 4]
```diff
@@ -1889,7 +1889,7 @@
         },
```
Comment (n8rzz) [inline package-lock.json]: hmm, interesting that these changed from `https` -> `http`.  

could you revert these changes by running: `git checkout origin/master package-lock.json` and then commit the result?

Comment (manuel3108) [reply package-lock.json]: Done, didn't see that one!
I think you mean `git checkout origin/develop package-lock.json`

Comment (n8rzz) [reply package-lock.json]: whoops!  yes, I did mean `develop`.  thank you!
